src/main/java/com/arpnetworking/metrics/impl/ApacheHttpSink.java

```diff
@@ -119,8 +127,14 @@ private ApacheHttpSink(final Builder builder) {
 
     private final class HttpDispatch implements Runnable {
 
-        /* package private */ HttpDispatch(final PoolingHttpClientConnectionManager connectionManager) {
-            _connectionManager = connectionManager;
+        /* package private */ HttpDispatch(
+                final PoolingHttpClientConnectionManager connectionManager,
+                final Duration dispatchErrorLoggingInterval) {
+                _connectionManager = connectionManager;
+             _dispatchErrorLogger = new RateLimitedLogger(
```

Comment:
That's no longer the case after we merged #13 which creates a single HttpDispatch instance for the entire worker pool. So while independent ApacheHttpSink instances will have independent rate limits within each there should be a singular rate limiter for the thread pool. Am I missing something here?